packages/address-book-controller/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Bump dependency on `@metamask/controller-utils` to ^5.0.0
```

Comment:
It would be nice to release this as well if we can, as it'd let us avoid having multiple copies of `@metamask/controller-utils` in our dependency tree. If we don't update this, we'll need v4 and v5 around, minimally.

Reply:
done